src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -40,9 +40,43 @@ The keyboard comes with Mac layout keys pre-installed for macOS which can also b
 |Switch support|Hot-swappable switches with 3 & 5 pin|
 |Rollover|N-key rollover|
 
-# How to change the backlight {% #how-to-change-the-backlight %}
+# Quick start  {% #quick-start %}
 
-Press `Fn + Home` to change the RGB backlight effect and cycle through all the lights.
+There are currently three modes to connect the Appwriter to the computer: wired, wireless, and Bluetooth.
+
+## Wired mode {% #wired-mode %}
+
+To use the wired mode, connect the USB-TYPE-C data cable from the keyboard to the computer and push the Appwriter switch towards the T.
+When connecting the keyboard for the first time using wired mode, your computer will install all required drivers on a successful connection.
+
+The Appwriter's battery will charge while connected using the wired mode. Make sure to use a charger that doesn't exceed a voltage/current = DSCV = 1A (max). Using a charger that exceeds this voltage and current ratings could easily damage your keyboard.
+
+## Wireless mode {% #wireless-mode %}
+
+Push the Appwriter switch towards the G if you want to use the wireless mode. After switching to wireless mode, long press the `FN` + `4` keys for around 3 seconds. After the `4` key light flashes quickly, plug the detached receiver into a computer USB port.
+
+## Bluetooth mode {% #bluetooth-mode %}
+
+To pair the keyboard in Bluetooth mode, push the switch to B and do a long press while pressing the `FN` + `1` keys. Once the `1` key flashes a fast light, you must search for the Appwriter in your Bluetooth devices to pair your devices.
```

Comment:
Based on the docs, the keyboard can be paired to 3 different devices. Short pressing FN + 1/2/3 will connect, while long pressing FN + 1/2/3 will pair. Let's make that clear in this section.

> do a long press

We say "long press" instead of "do a long press" elsewhere so let's use "long press" here too.

> do a long press while pressing the `FN` + `1` keys

Also, this is kind of confusing. Can you simplify it?